### PR TITLE
Formula renames and fix Circle CI

### PR DESCRIPTION
**Pull request by fjperini**
Preparing the formulas change.

and fixing the error in Circle CI when renaming them:

```
Persisting to Workspace

The specified paths did not match any files in /Users/distiller/project/bottles
```

### CODE REVIEW

File: Formula/formula_renames.json [hunk 1]
```diff
@@ -12,5 +12,8 @@
   "orfeo-54": "orfeo5",
   "qgis-214": "qgis2-ltr",
   "qt-mysql": "qt4-mysql",
-  "saga-gis": "saga-gis-lts"
+  "saga-gis": "saga-gis-lts",
+  "qgis2-ltr": "qgis-ltr@2.14",
```
Comment (3nids) [inline Formula/formula_renames.json]: why not also to 2.18?

Comment (fjperini) [reply Formula/formula_renames.json]: yes, this is just a test to see if Circle CI is fixed

Comment (3nids) [reply Formula/formula_renames.json]: ok, thanks!

Comment (fjperini) [reply Formula/formula_renames.json]: I'm in it, I'll make some changes.
It seems that it does not find the bottles in Circle CI, that's why it does not rename them.

**Comment by fjperini** [comment]
@3nids

**Pull request closed without merging**